boards/posix/native_posix/doc/index.rst

```diff
@@ -408,6 +413,42 @@ option ``-attach_uart_cmd=<"cmd">``. Where the default command is given by
 Note that the default command assumes both ``xterm`` and ``screen`` are
 installed in the system.
 
+.. _serial_ports:
```

Comment:
maybe better name it "_uart_native_tty" so the anchor name is a bit more unique (the text in the link can be different than the anchor it refers to)